niworkflows/anat/ants.py

```diff
@@ -863,6 +886,8 @@ def init_n4_only_wf(
     atropos_refine=True,
     atropos_use_random_seed=True,
     bids_suffix="T1w",
+    use_bspline_grid=False,
+    min_n4_iter=False,
```

Comment:
Do we use this workflow in nirodents? I don't think so, and I'd honestly avoid changing it at this moment.